ci/sonar-scanner.sh

```diff
@@ -23,7 +23,7 @@ args+=("-o:spatialos")
 args+=("-d:sonar.login=${TOKEN}")
 args+=("-d:sonar.project_key=spatialos_gdk-for-unity")
 args+=("-d:sonar.host.url=https://sonarcloud.io")
-args+=("-d:sonar.buildString=${BUILDKITE_MESSAGE}")
+args+=("-d:sonar.buildString=${BUILDKITE_MESSAGE:0:100}")
```

Comment:
Does it start from 0 or 1 with bash substrings?

Reply:
0